pkgs/top-level/python-packages.nix

```diff
@@ -14941,6 +14941,72 @@ in {
     };
   };
 
+  pythonnet = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
Done